user/configure-device.rst

```diff
@@ -50,7 +50,11 @@ the terminal.
 
     Continue with the procedures in
     `the OpenWRT virtualbox guide <https://openwrt.org/docs/guide-user/virtualization/virtualbox-vm>`_.
-    It's required to enable SSH access and connect the VM to the internet.
+    It's required to enable SSH access and connect the VM to the internet. Note that both Adapter1 and Adapter2 
+    use "Adapter Type: Intel PRO/1000 MT Desktop". Also, please do not use the same IP Address that you used for 
+    the local OpenWISP website you hosted before. That suggested change applies only when you boot into the OpenWRT 
+    device as per the description of the above link. (For example, if you set 192.168.56.2 as the IP Address of your
```

Comment:
move the final dot after `link` to after the closing parentheses, also change `For` to `for`